### ISSUE TITLE
rustdoc: with 1.80 throws invalid_codeblock_attributes warning

### ISSUE THREAD

**Issue by p-alik** [rust-lang/rust#128869]
I tried this code:

```rust
/// ``` // with rustdoc 1.80.{0,1} the comment causes rustdoc::invalid_codeblock_attributes warning
/// use foo::add;
/// assert_eq!(add(1, 2), 3);
/// ```
pub fn add(left: usize, right: usize) -> usize {
    left + right
}
```

I expected to see this happen: *no warning*
```
$ echo 1.79.0 > rust-toolchain 
$ rustdoc src/lib.rs 
```
Instead, this happened: *getting rustdoc::invalid_codeblock_attributes warning*
```
$ echo 1.80.0 > rust-toolchain 
$ rustdoc src/lib.rs 
warning: unexpected character `/`
 --> src/lib.rs:1:1
  |
1 | / /// ``` // with rustdoc 1.80.{0,1} the comment causes rustdoc::invalid_codeblock_attributes warning
2 | | /// use foo::add;
3 | | /// assert_eq!(add(1, 2), 3);
4 | | /// ```
  | |_______^
  |
  = note: `#[warn(rustdoc::invalid_codeblock_attributes)]` on by default

warning: 1 warning emitted
```
